server/src/constants.ts

```diff
@@ -34,6 +34,7 @@ export const geodataDatePath = join(GEODATA_ROOT_PATH, 'geodata-date.txt');
 export const geodataAdmin1Path = join(GEODATA_ROOT_PATH, 'admin1CodesASCII.txt');
 export const geodataAdmin2Path = join(GEODATA_ROOT_PATH, 'admin2Codes.txt');
 export const geodataCities500Path = join(GEODATA_ROOT_PATH, citiesFile);
+export const lockfilePath = process.env.IMMICH_BUILD_LOCK_PATH || './build-lock.json';
```

Comment:
Maybe this should just always use env, same as the other build-metadata stuff, with the base-image Dockerfile setting this particular one?

Reply:
IMO, it would be great to group `www/`, `build-lock.json` and `geodata/` into one folder and specify its path with an env. But I don't know if that would limit other packages like the Immich package from the AUR.

Reply:
I kind of really rate having separate env for every file that we read from disk. I'd be perfectly happy with bundling all these files into a single, folder and having a single env for it. Something like `IMMICH_BUILD_FILES` or something.

Reply:
I would be really happy too. Are you ok to have all these files/folders to `/usr/src/resources` ?
We would have:
- `/usr/src/resources/www`
- `/usr/src/resources/geodata`
- `/usr/src/resources/build-lock.json`

Reply:
Yup